mega.go

```diff
@@ -1037,6 +1037,112 @@ func (m *Mega) NewDownload(src *Node) (*Download, error) {
 	return d, nil
 }
 
+// Create a new public Download from the hash and key
```

Comment:
Where do you get the hash for a public file? From the URL?

What about directories?

Reply:
> Where do you get the hash for a public file? From the URL?

```
https://mega.nz/file/AAAAAA#BBBBBBBBB
                     ↑      ↑
                     │      └ Key
                     └ Hash
```

> What about directories?

I've not actually tested this with directories, but I'd guess that they work completely differently (looking at the URLs at least).